components/jwt-init/run-sidecar.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+/usr/local/bin/execute-get-token.sh &
+/usr/local/bin/execute-get-vault-secrets.sh
+RT=$?
+if [ "$RT" == "0" ]; then
```

Comment:
Hmm, looks like there might be a race condition here with `get-token` we may want to somehow give a leeway of backoff if it fails.